packages/fork-choice/src/forkChoice/interface.ts

```diff
@@ -149,14 +143,14 @@ export type OnBlockPrecachedData = {
    * powBlock = getPowBlock((block as merge.BeaconBlock).body.executionPayload.parentHash)
    * ```
    */
-  powBlock?: PowBlock;
+  powBlock?: merge.PowBlock;
```

Comment:
Why is this necessary? The fork-choice doesn't care about the root at all, so converting to bytes is useless work

Reply:
ok so I'll make 2 types:
+ PowBlock for the spec test
+ PowBlockHex for forkchoice